sklearn/svm/src/liblinear/liblinear_helper.c

```diff
@@ -87,10 +87,10 @@ static struct feature_node **csr_to_sparse(double *values,
 
         sparse[i] = malloc ((n+have_bias+1) * sizeof(struct feature_node));
         if (sparse[i] == NULL) {
-            int l;
-            for (l=0; l<i; l++)
-                free(sparse[l]);
-            break;
+            for (j=0; j<i; j++)
```

Comment:
nit: To stay consistent with the rest of the file:

```suggestion
            for (j=0; j<i; ++j)
```

Reply:
Done. By the way, you can see the changes between my original commit and the amended commit by clicking "force-pushed" in the pull request.